### PR TITLE
Adding Maven dependencies

### PR DESCRIPTION
**Pull request by anciuvaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,11 +2,49 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
+    <modelVersion>4.0.0</modelVersion>gi
```
Comment (euliancom) [inline pom.xml]: please remove `gi`

File: pom.xml [hunk 2]
```diff
@@ -2,11 +2,49 @@
...
+            <version>20200518</version>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.3.0</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 </project>
```
Comment (euliancom) [inline pom.xml]: please add a new line

**Review by euliancom — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (anciuvaga) [reply pom.xml]: Done 👍

File: pom.xml (hunk 2 above)
Comment (anciuvaga) [reply pom.xml]: Done 👍

**Review by euliancom — APPROVED** [review]

**Pull request merged**